### PR TITLE
Nixify / Allow the use of a global binary

### LINKED ISSUE #29: Document how to run this on nix
Maybe @JonathanLorimer could send a PR for the readme one day!

### PR DESCRIPTION
**Pull request by JonathanLorimer**
Fixes #29

### CODE REVIEW

File: ftplugin/agda.vim [hunk 1]
```diff
@@ -1,4 +1,9 @@
-call nvimhs#start(expand('<sfile>:p:h:h'), 'cornelis', ['-v', 'DEBUG', '-l', '/tmp/cornelis.log'])
+if exists("g:cornelis_use_global_binary")
+  call remote#host#Register('cornelis', '*', rpcstart('cornelis', []))
+else
```
Comment (JonathanLorimer) [inline ftplugin/agda.vim:3]: In the future we could allow people to provide the global binary path as a string, but for now this works for me!

Comment (isovector) [reply ftplugin/agda.vim:3]: Might be a good thing to tackle at the same time as https://github.com/isovector/cornelis/issues/12

Comment (JonathanLorimer) [reply ftplugin/agda.vim:3]: good call

**Review by JonathanLorimer — COMMENTED** [review]
Self Reviewed

**Comment by isovector** [comment]
Thanks so much!

**Pull request merged**